spec/text-editor-spec.coffee

```diff
@@ -160,15 +160,15 @@ describe "TextEditor", ->
   describe "title", ->
     describe ".getTitle()", ->
       it "uses the basename of the buffer's path as its title, or 'untitled' if the path is undefined", ->
-        expect(editor.getTitle()).toBe 'sample.js'
+        expect(editor.getTitle(atom.workspace.getTextEditors())).toBe 'sample.js'
```

Comment:
Do you think we could avoid passing `atom.workspace.getTextEditors()` here? Seems like the (short) title shouldn't need to refer to other editors.